### DISCUSSION TITLE
How to modify the content?

### DISCUSSION THREAD

**Discussion by nironixon** [ThreeMammals/Ocelot#1936] [Q&A]
## New Feature / Existing Feature?
Hi there, ALL new to Ocelot but seems very promising!

I need to transform the json response from a upstream server to better fit the capabilities of a given client. Is it possible to inject some kind of callback for a given route so i can modify the content?

**Comment by raman-m** [comment]
@nironixon  Hi Niclas!
**Yes, it is!** ... by developing of a custom [Delegating Handler](https://ocelot.readthedocs.io/en/latest/features/delegatinghandlers.html).

@ggnaegi
Considering our latest changes in request mapper, and content streaming... Is it possible to transform content?
Seems the author should create absolutely new `HttpRequestMessage` object to forward, not transforming the current `HttpRequestMessage request` argument.
